packages/editor/src/components/rich-text/list-edit.js

```diff
@@ -35,27 +35,39 @@ function isActiveListType( editor, tagName, rootTagName ) {
 	return list.nodeName.toLowerCase() === tagName;
 }
 
-export const ListEdit = ( { editor, onTagNameChange, tagName } ) => (
+export const ListEdit = ( { editor, onTagNameChange, tagName, onSyncDOM } ) => (
 	<Fragment>
 		<RichTextShortcut
 			type="primary"
 			character="["
-			onUse={ () => editor.execCommand( 'Outdent' ) }
+			onUse={ () => {
+				editor.execCommand( 'Outdent' );
```

Comment:
In all the calls to editor.execCommand we need to perform the onSyncDom right? Would it make sense to have a function in RichText called editorExecCommand that abstracts this?

Reply:
The calls are and should be limited to this file, so I would prefer not to. Ideally we should remove these calls soon.